home-application/src/main/java/com/softserveinc/ita/homeproject/application/exception/mapper/jax/BadRequestExceptionMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.softserveinc.ita.homeproject.application.exception.mapper.jax;
+
+import com.softserveinc.ita.homeproject.application.exception.mapper.BaseExceptionMapper;
+import cz.jirutka.rsql.parser.RSQLParserException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class BadRequestExceptionMapper extends BaseExceptionMapper<RSQLParserException> {
```

Comment:
The name of a mapper should be after the exception it maps
RSQLParserExceptionMapper